x/evm/config/configurator.go

```diff
@@ -50,6 +64,16 @@ func (ec *EVMConfigurator) Configure() error {
 		return fmt.Errorf("error configuring EVMConfigurator: already sealed and cannot be modified")
 	}
 
+	if ec.chainConfig != nil {
```

Comment:
in which case would this be nil?

Reply:
if the configurator didnt set a specific chain config, we use the default configuration